package/MDAnalysis/coordinates/GRO.py

```diff
@@ -269,19 +275,35 @@ def write(self, selection, frame=None):
            resName and atomName are truncated to a maximum of 5 characters
         """
         # write() method that complies with the Trajectory API
-        u = selection.universe
-        if frame is not None:
-            u.trajectory[frame]  # advance to frame
-        else:
-            frame = u.trajectory.ts.frame
 
-        # make sure to use atoms (Issue 46)
-        atoms = selection.atoms
-        # can write from selection == Universe (Issue 49)
-        coordinates = atoms.positions
+        # if frame is not None:
+        #     u.trajectory[frame]  # advance to frame
+        # else:
+        #     frame = u.trajectory.ts.frame
+
+        try:
```

Comment:
@richardjgowers [This](https://github.com/MDAnalysis/mdanalysis/pull/1196/files#diff-221626a4bf4542baa2eeb062fb0d4368L369) line is causing the problem. AtomGroup doesn't have a triclinic dimensions. What would be a good way to handle this? I could wrap the if-else block in another if-else but that doesn't seem good.